src/prefect/environments/execution/k8s/job.py

```diff
@@ -39,7 +40,9 @@ class KubernetesJobEnvironment(Environment, _RunMixin):
     `$ /bin/sh -c "python -c 'import prefect; prefect.environments.execution.load_and_run_flow()'"`
 
     Args:
-        - job_spec_file (str, optional): Path to a job spec YAML file
+        - job_spec_file (str, optional): Path to a job spec YAML file.
+            When this class is initialized, the contents of `job_spec_file` are read in
+            and stored in `self._job_spec`.
```

Comment:
I don't think we need to document the internals of this class - noting that the file should be on the machine that registers the flow, not the machine that runs the flow should be enough.

Reply:
Ok got it, I can reword it.

Reply:
I changed the wording in https://github.com/PrefectHQ/prefect/pull/2950/commits/2ea3a06eeb973ec036e034c871c66738d45375e1